### ISSUE TITLE
🦄  We need to investigate which role drives the foreign-source and requisition creation in OpenNMS Horizon

### ISSUE THREAD

**Issue by indigo423** [opennms-forge/ansible-provisioning#13]
Before we can create a Node in OpenNMS Horizon we need a requisition + foreign-source in OpenNMS Horizon. We have to create that before we can run roles or tasks for each deployment.

**Comment by indigo423** [comment]
Before we can create a Node in OpenNMS Horizon we need a requisition + foreign-source in OpenNMS Horizon. We have to create that before we can run roles or tasks for each deployment.

**Comment by mfuhrmann** [comment]
This is already covered by https://github.com/opennms-forge/ansible-provisioning/blob/main/ansible/roles/horizon-provision/tasks/main.yml#L2-L47 and defining the corresponding variables.
